oalab/src/openalea/oalab/plugins/controls/widgets.py

```diff
@@ -47,6 +47,169 @@ def value(self, interface=None):
         return self.text()
 
 
+class QFloatSlider(QtGui.QSlider):
+
+    floatValueChanged = QtCore.Signal(float)
+
+    def __init__(self, orientation=QtCore.Qt.Horizontal):
+        QtGui.QSlider.__init__(self, orientation)
+        self.connect(self, QtCore.SIGNAL('valueChanged(int)'), self.notifyValueChanged)
```

Comment:
Just for information, you can also use this notation to connect signals:
self.valueChanged.connect(self.notifyValueChanged)